test/functional/setup/create_logins_azure.sql

```diff
@@ -0,0 +1,18 @@
+--for this script to work in Azure, use sqlcmd to connect to master database 
+IF NOT EXISTS (SELECT name FROM sys.sql_logins WHERE name = 'test_password')
+BEGIN
+    CREATE LOGIN test_password WITH PASSWORD='! ;4triou';
```

Comment:
@yitam can we drop the test_password, and create it again? and at the end of the test, delete the pass too